kodilla-spring/src/test/java/com/kodilla/spring/forum/ForumUserTestSuite.java

```diff
@@ -0,0 +1,26 @@
+package com.kodilla.spring.forum;
+
+import org.junit.jupiter.api.Test;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.context.ApplicationContext;
+import org.springframework.context.annotation.AnnotationConfigApplicationContext;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+@SpringBootTest
```

Comment:
ta adnotacje jest potrzebna, jesli uzywasz `@Autowire` to wyciagniecia beanow. W testcie jest `new AnnotationConfig...` wiec `@SpringBootTest` jest niepotrzebne

Reply:
w KodillaSpringApplicationTests analogicznie